### ISSUE TITLE
fixmes in weighting (Trac #1237)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1231]
python/fluxes.py:	FIXME: this predicts a flux that is a factor of ~2 larger than Hoerandel
python/weighting.py:				# FIXME: more careful checking of the parameters

Migrated from https://code.icecube.wisc.edu/ticket/1237
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "python/fluxes.py:\tFIXME: this predicts a flux that is a factor of ~2 larger than Hoerandel\npython/weighting.py:\t\t\t\t# FIXME: more careful checking of the parameters", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo core", 
    "summary": "fixmes in weighting", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-20T08:27:35", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150921T12:16:25: 
* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 137791:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137791"
> Fix FIXMEs, mostly with the fixing hammer. Fixes #1231.
> 
> Other words on the same dictionary page: fixed, fixation, fixative, fixture.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
